opentelemetry-sdk/src/metrics/periodic_reader.rs

```diff
@@ -251,16 +251,16 @@ impl<E: PushMetricExporter> PeriodicReader<E> {
                             // tells that it timed out.
                             if export_result.is_err() || shutdown_result.is_err() {
                                 if response_sender.send(false).is_err() {
-                                    otel_info!(
+                                    otel_debug!(
```

Comment:
Would like to discuss more on this. The code comment above explains why this is Info. See if you agree with this.

Again, this should not occur in normal workflows so should not cause cogs issues.

Reply:
My reading of the comment was that while I understand it's an unexpected situation, it's not entirely clear that it's actionable. If the shutdown-calling thread went away, presumably they no longer care about this?

Reply:
This is actionable to service owners: If they see this log often, it is an indication to them that they need to increase their timeouts.

(Nothing they can do immediately as the shutdown() is already over, and there is no option to try-again)

Reply:
Changed back